core/spec/models/spree/shipping_rate_spec.rb

```diff
@@ -38,20 +38,21 @@
         end
       end
 
-      context "when the tax rate is from a non-default zone" do
-        let!(:default_zone) { create(:zone, :default_tax => true) }
+      context "when the tax rate is from another zone" do
         let!(:non_default_zone) { create(:zone, :default_tax => false) }
-        let(:tax_rate) do
+
+        let(:non_default_tax_rate) do
           create(:tax_rate,
             :name => "VAT",
-            :amount => 0.1,
+            :amount => 0.2,
             :included_in_price => true,
             :zone => non_default_zone)
         end
-        before { shipping_rate.tax_rate = tax_rate }
+        before { shipping_rate.tax_rate = non_default_tax_rate }
 
-        it "shows correct tax amount" do
-          expect(shipping_rate.display_price.to_s).to eq("$10.00 (excl. $0.91 #{tax_rate.name})")
+        pending "I would expect this to be analogue to line items: First calculate the net price, then add the foreign VAT.\n"+
```

Comment:
Bear in mind that this is the edge case for charging shipping costs for downloads. Anyone ever paid for a being able to download something, rather than have it shipped?